index.html

```diff
@@ -74,16 +81,18 @@ <h4>Canopy</h4>
                   <li class="prog-lang">Javascript</li>
               </ul>
               <button>See Project</button>
-          </div> 
-        </div>  
+              </div>          
+          </div>
+        </div>
 
 
         <!--Work Section 2-->
 
         <div class="work-container">
           <div class="img img-2">
-            <img src="./images/Snapshoot2.png" alt="second-work">
-            <h2>Multi-Post</h2>
+            <img src="./images/Desktop/Grid-2/Snapshoot_Portfolio_2.svg.svg" alt="second-work">
+            <div class="content">
+              <h2>Multi-Post</h2>
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly make sure your design matches that of your preferred template😉
- Your cards are not following the design on the Figma template🙄, I'd suggest you use 👇
```
work-container:nth-child(even) .cards {
    flex-direction: row-reverse !important;
}
```
to reverse the card content and image as seen in the original design🤔
- Figma design
![screenshot (53)](https://user-images.githubusercontent.com/44624138/144599975-ef8d7e27-20c4-4516-b55e-4a90d1376cd6.png)
- Your design
![screenshot (52)](https://user-images.githubusercontent.com/44624138/144600023-b9563df4-2da7-4d04-8fcf-8f92cfeb2cf0.png)

Reply:
- [ ] Kindly fix the broken image on your card too for a better user interface👌

Reply:
I accidentally missed the image link....... Now I added it